instances.go

```diff
@@ -108,6 +143,23 @@ func (c *Client) getInstancePolicy(ctx context.Context, policyType string, polic
 	return err
 }
 
+// GetMetricsInstancePolicy retrieves the metrics policy details associated with the instance
+// For more information can refer the Key Protect docs in the link below:
+// https://cloud.ibm.com/docs/key-protect?topic=key-protect-manage-metrics
```

Comment:
This URL does not exist. Coordinate with Kierra or Robert regarding documentation.

Reply:
Currently it does not exist, Kierra provided me with this URL as the probable one

Reply:
That's good for me.